UltimateWoltlabMaster.sh

```diff
@@ -0,0 +1,123 @@
+reset
+
+echo "UltimateWoltlabMaster von Justman10000"
+echo "Dies ist unter der GNU General Public License v3.0 lizenziert"
+
+echo "Was soll getan werden?"
+echo "Installieren [1]"
+echo "Deinstallieren [2]"
+echo "Zurück [3]"
+read WhatDo
+
+clear
+
+FileName=WoltlabSuite.zip
+
+case $WhatDo in
+    "1")
+        wget https://assets.woltlab.com/release/woltlab-suite-5.4.10.zip
+        sleep 1
+        mv woltlab-suite-5.4.10.zip $FileName
+        sleep 1
+        unzip $FileName
+        sleep 1
+        rm -r *.txt
+        sleep 1
+
+        clear
+
+        echo "In welchem Pfad soll WoltLab installiert werden?"
```

Comment:
The texts are only in German. However, since the WSC is multilingual (at least German & English by default), English should also be available.